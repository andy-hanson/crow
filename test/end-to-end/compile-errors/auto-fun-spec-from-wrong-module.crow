no-std
import
	crow/bool: bool
	crow/misc: new, void

main void()
	()

equal[t] spec
	== bool(a t, b t)

r record
== bool(a r, b r)
