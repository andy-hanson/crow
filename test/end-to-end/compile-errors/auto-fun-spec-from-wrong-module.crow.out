exit code: 0
stderr:
test/end-to-end/compile-errors/auto-fun-spec-from-wrong-module.crow 13:1-13:3 Spec for automatic function comes from unexpected module.
